### DISCUSSION TITLE
ESP32 DevKitC NodeMCU auch möglich?

### DISCUSSION THREAD

**Discussion by getBKs** [frickelzeugs/FingerprintDoorbell#109] [Q&A]
Hallo,

ich habe mir eine D1 mini besorgt, das Board scheint defekt zu sein, ich konnte es nicht flashen. Angeschlossen war es über den USB-Port des PCs. Der kleine IC gleich neben der USB-Buchse am D1 wurde extrem heiß.

Ich hatte noch einige ESP32 DevBoards da uns habe das darauf geflasht. Ging absolut Problemlos. Ich kann in die Einstellungen usw.

Nur wie ist da die Pinbelegung, ich nehme an anders. Die sind ja auch nicht identisch beschriftet.

Wäre es möglich dieses ESP-Board zu nutzen?

Freue mich auf Infos und bin absolut aufgeregt, mit Fingerabdruck in die Wohnung zu kommen.

**Comment by tilly001** [comment]
![Screenshot_20241219_211659_Chrome](https://github.com/user-attachments/assets/3805b8a9-d131-402f-abd1-ea08e05c3a36)
Hi getBKs,

Ich nutze das Board auch. Geht ohne Probleme.

**Comment by getBKs** [comment]
Hi tilly001,

danke für den goldenen Tipp.
Ist es denn nicht so, dass man mit diesem Board dann keine Spannungswandler braucht, wenn es mit 5V über USB angeschlossen wird? Die 3.3V für den Sensor werden ja vom Board bereits ausgegeben.

**Reply by tilly001** [reply]
Genau, ich betreibe es mit einem normalen USB Netzteil über 5V. Und wenn du es einfach anschließen willst gönnst du dir noch die hier: 
![Screenshot_20241223_013113_Amazon Shopping](https://github.com/user-attachments/assets/e4c1d66a-d1a1-44a9-b9a6-15babece2cb6)

**Comment by getBKs** [comment]
Fantastisch. Nochmal vielen Dank für die Infos.

Jetzt muss ich es noch schaffen, das blaue Leuchten auszuschalten. Gibt ja bereits eine Diskussion, die es geschafft haben. Muss nur noch verstehen, wie :)

**Reply by tilly001** [reply]
https://github.com/mickler2021/FingerprintDoorbell/releases/tag/v0.4.1

Installiere mal dieses Update ... Da kannst du einiges mehr bzgl. der Ring Farben steuern.

**Comment by getBKs** [comment]
Du bist ja Maschine :) Natürlich geht hier auch ein großer Lob an den Entwickler
